<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationManager</class>
 <widget class="QDialog" name="ConfigurationManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CopyQ Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_clipboard">
        <attribute name="icon">
         <iconset resource="copyq.qrc">
          <normaloff>:/images/tab_clipboard.svg</normaloff>:/images/tab_clipboard.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>C&amp;lipboard</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>S&amp;ynchronize</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QCheckBox" name="checkBoxClip">
                  <property name="toolTip">
                   <string>Save clipboard in history.</string>
                  </property>
                  <property name="text">
                   <string>clipboard &gt;&gt; history (&amp;1)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxCopyClip">
                  <property name="toolTip">
                   <string>Whether to copy clipboard content to the primary selection.</string>
                  </property>
                  <property name="text">
                   <string>clipboard &gt;&gt; selection (&amp;2)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxSel">
                  <property name="toolTip">
                   <string>Save primary selection in history.</string>
                  </property>
                  <property name="text">
                   <string>selection &gt;&gt; history (&amp;3)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxCopySel">
                  <property name="toolTip">
                   <string>Whether to copy primary selection content to the clipboard.</string>
                  </property>
                  <property name="text">
                   <string>selection &gt;&gt; clipboard (&amp;4)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxConfirmExit">
             <property name="toolTip">
              <string>Confirm application exit.</string>
             </property>
             <property name="text">
              <string>Confirm application e&amp;xit</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_general">
        <attribute name="icon">
         <iconset resource="copyq.qrc">
          <normaloff>:/images/tab_history.svg</normaloff>:/images/tab_history.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>Clipboard &amp;History</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Maximum &amp;number of items in history:</string>
               </property>
               <property name="buddy">
                <cstring>spinBoxItems</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxItems">
               <property name="toolTip">
                <string>Maximum number of items in history.</string>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>200</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>N&amp;umber of items in tray menu:</string>
               </property>
               <property name="buddy">
                <cstring>spinBoxTrayItems</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxTrayItems">
               <property name="toolTip">
                <string>Number of items in tray menu.</string>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Maximum image size (set to 0 for original size):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>&amp;Width:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>spinBoxImageWidth</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxImageWidth">
               <property name="toolTip">
                <string>Maximum width of image displayed in history.</string>
               </property>
               <property name="maximum">
                <number>4096</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&amp;Height:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>spinBoxImageHeight</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxImageHeight">
               <property name="toolTip">
                <string>Maximum height of image displayed in history.</string>
               </property>
               <property name="maximum">
                <number>4096</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>External &amp;editor command (%1 is file to edit):</string>
               </property>
               <property name="buddy">
                <cstring>lineEditEditor</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditEditor">
               <property name="toolTip">
                <string>External editor command (%1 is file to edit).</string>
               </property>
               <property name="text">
                <string>gedit %1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>&amp;Format display priority:</string>
               </property>
               <property name="buddy">
                <cstring>plainTextEditFormats</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEditFormats">
               <property name="toolTip">
                <string>List of clipboard mime types that will be stored in history (in given display order).</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_commands">
        <attribute name="icon">
         <iconset resource="copyq.qrc">
          <normaloff>:/images/tab_commands.svg</normaloff>:/images/tab_commands.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>Co&amp;mmands</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QListWidget" name="listWidgetCommands">
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="rightMargin">
                <number>20</number>
               </property>
               <item>
                <widget class="CommandWidget" name="widgetCommand" native="true">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::WheelFocus</enum>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButtoAdd">
               <property name="text">
                <string>Add [+]</string>
               </property>
               <property name="shortcut">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonRemove">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Remove [Del]</string>
               </property>
               <property name="shortcut">
                <string>Del</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonUp">
               <property name="text">
                <string>Move Up [Ctrl+Up]</string>
               </property>
               <property name="shortcut">
                <string>Ctrl+Up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonDown">
               <property name="text">
                <string>Move Down [Ctrl+Down]</string>
               </property>
               <property name="shortcut">
                <string>Ctrl+Down</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxCommands">
               <item>
                <property name="text">
                 <string>Insert Default Command</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Commands can be used to run an &lt;span style=&quot; font-weight:600;&quot;&gt;application&lt;/span&gt; on new clipboard content or &lt;span style=&quot; font-weight:600;&quot;&gt;banning the item from history list&lt;/span&gt;. If clipboard text is matched by regular expression in &lt;span style=&quot; font-style:italic;&quot;&gt;Match&lt;/span&gt; text field the appropriate action is taken.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_shortcuts">
        <attribute name="icon">
         <iconset resource="copyq.qrc">
          <normaloff>:/images/tab_shortcuts.svg</normaloff>:/images/tab_shortcuts.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>&amp;Shortcuts</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Sh&amp;ow/hide clipboard history</string>
               </property>
               <property name="buddy">
                <cstring>pushButton</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton">
               <property name="toolTip">
                <string>Shortcut to show or hide clipboard history.</string>
               </property>
               <property name="text">
                <string>(No Shortcut)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Show tray &amp;menu</string>
               </property>
               <property name="buddy">
                <cstring>pushButton_2</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButton_2">
               <property name="toolTip">
                <string>Shortcut to show tray menu.</string>
               </property>
               <property name="text">
                <string>(No Shortcut)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_appearance">
        <attribute name="icon">
         <iconset resource="copyq.qrc">
          <normaloff>:/images/tab_appearance.svg</normaloff>:/images/tab_appearance.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>&amp;Appearance</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>&amp;Style sheet (leave empty to reset):</string>
             </property>
             <property name="buddy">
              <cstring>plainTextEdit_css</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_css">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowserStyleHelp">
           <property name="lineWrapMode">
            <enum>QTextEdit::NoWrap</enum>
           </property>
           <property name="openLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandWidget</class>
   <extends>QWidget</extends>
   <header>commandwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBoxClip</tabstop>
  <tabstop>checkBoxCopyClip</tabstop>
  <tabstop>checkBoxSel</tabstop>
  <tabstop>checkBoxCopySel</tabstop>
  <tabstop>checkBoxConfirmExit</tabstop>
  <tabstop>spinBoxItems</tabstop>
  <tabstop>spinBoxTrayItems</tabstop>
  <tabstop>spinBoxImageWidth</tabstop>
  <tabstop>spinBoxImageHeight</tabstop>
  <tabstop>lineEditEditor</tabstop>
  <tabstop>plainTextEditFormats</tabstop>
  <tabstop>listWidgetCommands</tabstop>
  <tabstop>widgetCommand</tabstop>
  <tabstop>pushButtoAdd</tabstop>
  <tabstop>pushButtonRemove</tabstop>
  <tabstop>pushButtonUp</tabstop>
  <tabstop>pushButtonDown</tabstop>
  <tabstop>comboBoxCommands</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="copyq.qrc"/>
 </resources>
 <connections/>
</ui>
